plugins/events_history_plugin/src/events_history/mgr/events_history_mgr.py

```diff
@@ -65,12 +74,14 @@ def parse_event_log_files(self):
             files.append(EventsConstants.EVENT_LOG)
             for file in files:
                 fname = os.path.join(EventsConstants.EVENT_LOGS_DIRECTORY, file)
+                Logger.log_message(f"Parsing file {fname}", LOG_LEVELS.INFO)
```

Comment:
If file not exists this message is incorrect. Message should be moved under the if, but need to mention list of files you are going to parse.